src/azure-cli-core/azure/cli/core/extensions/experimental.py

```diff
@@ -60,7 +60,7 @@ def use_skeleton(_, event_data):
         try:
             args = event_data['args']
             skeleton_index = args.index('--use-skeleton')
-            skeleton_value = args[skeleton_index + 1]  # pylint: disable=unused-variable
+            skeleton_value = args[skeleton_index + 1]  # noqa, pylint: disable=unused-variable
```

Comment:
As far as I know, the skeleton feature is not complete and isn't used currently.